Gemfile

```diff
@@ -51,6 +50,14 @@ gem 'sidekiq-statsd', '0.1.5'
 
 gem 'deprecated_columns', '0.1.0'
 
+if ENV['GDS_API_ADAPTERS_DEV']
```

Comment:
Sorry, I didn't explain myself at all. The git ref is absolutely fine.

The thing I personally dislike is the ENV variable as it means you need to read the Gemfile.lock to figure out which thing is being used at any one time. I've been bitten in the past where I've checked in a Gemfile.lock pointing at a local repo. Having said all that, it seems that this pattern's usefulness outweighs the potential trip hazard for you. Does anyone else have an opinion on this?